tests/integration/records/single_db/base_records_test.py

```diff
@@ -29,7 +29,10 @@
 logger = logging.getLogger(__name__)
 
 
-set_stream_logging()
+set_stream_logging(level=logging.DEBUG)
+logging.getLogger('botocore').setLevel(logging.INFO)
+logging.getLogger('boto3').setLevel(logging.INFO)
+logging.getLogger('urllib3').setLevel(logging.INFO)
```

Comment:
Looks like you did some grunt work to find the noisier external libraries so you could chase down DEBUG-level messages in Records Mover.   It might be worth rolling control of those into a set_stream_logging() parameter at some point, or adding a comment at least describing the getLogger() syntax.  Not everyone using this is an experienced Pythonista and this is a good signal they might have this need as well.

(also just a reminder that the volume is still turned up on these integration tests)

Reply:
In this case the normal behavior of set stream logging appears to be correct-- it's just setting the records_mover logger. I couldn't quite figure out what was making the root logger DEBUG-- maybe something in nose itself? Whatever it is, I couldn't find it-- but this worked to make it less noisy.